iree/compiler/Codegen/Utils/Utils.h

```diff
@@ -42,22 +56,6 @@ IREE::HAL::ExecutableEntryPointOp getEntryPoint(FuncOp funcOp);
 /// formation to tile and distribute the ops.
 SmallVector<unsigned> getPartitionedLoops(Operation *op);
 
-/// Returns the untiled type of a tiled view for both tensor and memref
-/// types. Either walks the `ViewOpInterface` chain (for memrefs) or the
-/// `subtensor` op chain (for tensors).
-Type getUntiledType(Value tiledView);
```

Comment:
I thought this is used somewhere. If it isnt, then great!

Reply:
Yeah, I spotted no users for it. Probably there was originally and various rounds of code changes killed them.